.github/workflows/conda-cpp-build.yaml

```
@@ -121,6 +121,19 @@ jobs:
         env:
           STEP_NAME: "C++ build"
           GH_TOKEN: ${{ github.token }}
+      - name: Get Package Name and Location
+        run: | 
+          echo "RAPIDS_PACKAGE_NAME=$(RAPIDS_NO_PKG_EXTENSION=true rapids-package-name conda_cpp)" >> "${GITHUB_OUTPUT}"
+          echo "CONDA_OUTPUT_DIR=$(echo $RAPIDS_CONDA_BLD_OUTPUT_DIR)" >> "${GITHUB_OUTPUT}"
```

```suggestion
          echo "CONDA_OUTPUT_DIR=${RAPIDS_CONDA_BLD_OUTPUT_DIR}" >> "${GITHUB_OUTPUT}"
```

These changes should be equivalent